src/components/Apps/AccountListTable/const.js

```diff
@@ -129,14 +129,16 @@ export const accountOtherActions = (vm) => [
   },
   {
     name: 'SecretHistory',
-    // 密文历史
     title: vm.$t('HistoryPassword'),
     can: () => vm.$hasPerm('accounts.view_accountsecret'),
     type: 'primary',
     callback: ({ row }) => {
+      vm.account = row
       vm.currentAccountColumn = row
-      vm.$nextTick(() => {
-        vm.showPasswordHistoryDialog = true
+      vm.showViewSecretDialog = false
+      vm.secretUrl = `/api/v1/accounts/account-secrets/${row.id}/`
+      setTimeout(() => {
+        vm.showViewSecretDialog = true
       })
     }
   },
```

Comment:
It looks like there's only a single code line with an extra blank line at the end of this file, which seems unrelated to your request. There doesn't appear to be any coding inconsistencies, bugs, or potential optimizations needed within this snippet provided.

The most relevant information you could extract from it is:

* A function export named `accountOtherActions()`.
* This function has two parameters being passed through it (`vm`).
* It returns `undefined`.

For further questions about JavaScript code (including debugging and testing strategies), feel free to ask!